CMakeLists.txt

```diff
@@ -210,6 +217,7 @@ if(NOT WIN32)
         core_optional_dep(LibDRM ENABLE_MIR)
       else()
         core_optional_dep(X ENABLE_X11)
+        core_optional_dep(XRandR ENABLE_X11)
```

Comment:
not directly but more by the way the platform display module selection flow is implemented (the messy one).

if i dont add the XRandR module then i get the error:
`build/windowing/windowing.a(WinEventsX11.cpp.o):WinEventsX11.cpp:function CWinEventsX11Imp::MessagePump(): error: undefined reference to 'XRRUpdateConfiguration'
` and two more XR* undefined references due to missing 

rockchip doesnt support opengl but only opengles and this is probably not usual. i guess most kodi platforms that have opengles also have opengl (e.g. most mesa based implementations do) and then according the selection flow XRandR is automatically added (about 10 lines before, line #209) or the old ./configure way is used to specify this explicitly when building (i used to do this before using cmake)

it seems strange to add the X module and not the XRandR module. might actually be best to merge the FindX.cmake and FindXRandR.cmake into one.

so please merge.